### PR TITLE
🚮 Remove versioning.json from this repo

### PR DESCRIPTION
**Pull request by danielrozenberg**
This file is currently unused in this repo, instead it's equivalent in https://github.com/ampproject/cdn-configuration will be used

### CODE REVIEW

**Comment by samouri** [comment]
Are we confident no other repos depend on this file? I.e. amp-toolbox? I have no reason to think they do, just cautioning as I've unwittingly broken dependent repos recently

**Comment by danielrozenberg** [comment]
I added this file quite recently, I don't believe anyone started using it

File: .github/workflows/sync-versioning-to-cdn.yml [hunk 1]
```diff
@@ -1,42 +0,0 @@
...
-
-      - name: ⭐ Sync versioning.json to the CDN ⭐
-        run: |
-          for CHANNEL in $(jq 'keys[]' build-system/global-configs/versioning.json --raw-output)
-          do
-            RTV=$(jq ".[\"${CHANNEL}\"]" build-system/global-configs/versioning.json --raw-output)
-            if [[ "${RTV}" != "null" ]]
-            then
-              echo "Setting ${CHANNEL} to ${RTV}"
-              wrangler kv:key put "${CHANNEL}" "${RTV}" --config .github/workflows/wrangler.toml --binding RTV
```
Comment (estherkim) [inline .github/workflows/sync-versioning-to-cdn.yml:34]: Can `wrangler.toml` be deleted too, or deleted later?

**Review by estherkim — APPROVED** [review]

File: .github/workflows/sync-versioning-to-cdn.yml (hunk 1 above)
Comment (danielrozenberg) [reply .github/workflows/sync-versioning-to-cdn.yml:34]: It's still used by `.github/workflows/sync-client-side-experiments-config-to-cdn.yml` so I'll delete it with the PR that removes the client-side experiments from this repo

**Pull request merged**